addons/mail/static/src/core/common/message.js

```diff
@@ -511,4 +517,32 @@ export class Message extends Component {
             message: this.props.message,
         });
     }
+
+    // TODO KFR: Create something like a translation data object
+    async attemptTranslation() {
+        const response = await this.rpc("/discuss/translation/translate", {body: this.message.body})
+        if (response?.value) {
+            this.message.translatedBody = markup(response.value);
+            this.message.translationSource = response.source;
+        }
+        else {
+            this.message.translationErrors = true;
+        }
```

Comment:
Nit-picking: `response` is a bit too broad for naming, something more meaningful like `translation` makes code slightly easier to read at a glance. In particular, because if you read `markup(response.value)` it forces you to read `response =`, whereas with `translation` you can get the context without necessarily having to read previous LOCs.

Reply:
Totally agree.